CONTRIBUTORS.txt

```diff
@@ -97,6 +108,7 @@ needs to be listed here.
 - Richard Low <ral@apple.com>
 - Romain Pouclet <rpouclet@apple.com>
 - Rukshani Athapathu <arukshani@users.noreply.github.com>
+- Rémi Doreau <32459935+ayshiff@users.noreply.github.com>
```

Comment:
@ayshiff / @arukshani  if you'd like a proper email address here, please let me know.

Reply:
remi.d45@gmail.com, thanks!

Reply:
thanks, done.